python/demo/nessie-delta-demo.ipynb

```diff
@@ -31,7 +31,7 @@
     "                    .config(\"spark.hadoop.fs.defaultFS\", 'file://' + os.getcwd() + '/spark_warehouse') \\\n",
     "                    .config(\"spark.hadoop.nessie.uri\", \"http://localhost:19120/api/v1\") \\\n",
     "                    .config(\"spark.hadoop.nessie.ref\", \"main\") \\\n",
-    "                    .config(\"spark.hadoop.nessie.auth_type\", \"NONE\") \\\n",
+    "                    .config(\"spark.hadoop.nessie.auth-type\", \"NONE\") \\\n",
```

Comment:
The demos can't change till the related change has been shipped to iceberg and released

Reply:
yes I know, but given that we don't run those demos automatically in this folder, there should be only a short time-window where this isn't working, since we could get this change into Iceberg this week

Reply:
If its in `main` it has to work.

Reply:
ok reverted the changes on the demos until we get a new iceberg release